internal/pkg/skuba/addons/gangway.go

```diff
@@ -125,6 +125,7 @@ metadata:
   labels:
     app: oidc-gangway
 spec:
+  revisionHistoryLimit: 3
```

Comment:
Already in https://github.com/SUSE/skuba/blob/c8998e8/internal/pkg/skuba/addons/gangway.go#L139